CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Master (Unreleased)
 
 - Fix an infinite loop error when `RSpec/ExcessiveDocstringSpacing` finds a description with non-ASCII leading/trailing whitespace. ([@bcgraham])
+- Add `RSpec/SpecFilePathSuffix` cop. ([@ydah])
+- Add `RSpec/SpecFilePathFormat` cop. ([@ydah])
+- Deprecate `RSpec/FilePath` cop. Split `RSpec/FilePath` into `RSpec/SpecFilePathSuffix` and `RSpec/SpecFilePathFormat`. ([@ydah])
```

Comment:
Those three lines are excessive. Just the “Split” sentence is fine.
My huge respect to your efforts is not measured by the number of lines in the changelist, @ydah !

Reply:
I had separated each commit in stages, so I also separated the changelog, but three lines is certainly an excessive. I updated it 👍